ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TK:
+  description: "Tank water system used to store water"
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+
+WSTC:
+  description: " Waste Compactor system used for compacting organic waste essential for conversion into manure."
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+
+PMP:
+  description: "Tag for pumps. Pumps move liquid (water, glycol, liquid CO2, etc.) from one part of a system
+    to another.Used for (irrigation,domestic purposes) different usecase then HVAC."
```

Comment:
nit: space

Reply:
Thanks you for correcting .Done the changes